app.pianocheetah.pianocheetah.json

```diff
@@ -0,0 +1,31 @@
+{
+   "app-id":  "app.pianocheetah.pianocheetah",
+   "command": "pianocheetah",
+   "sdk":     "org.kde.Sdk",
+   "runtime": "org.kde.Platform",
+   "runtime-version": "6.6",
+   "modules": [
+      {
+         "name": "cp_to_app",
+         "buildsystem": "simple",
+         "build-commands": [
+            "cp -pr share /app",
+            "cp -pr bin   /app"
+         ],
+         "sources": [
+            {
+               "type": "dir",
+               "path": "_build/files"
```

Comment:
So how is this going to build? You have to use something that uses a url.

Reply:
well my app is closed source.  I build it with flatpak-build from my source into a build directory I call _build.  Then when I make the repo, It'll just use that _build directory and make the repo by copying _build/files/bin and share to /app 

So I end up with the repo having /app/bin and /app/share
 
Which is all this manifest is doing.  
  
I told a guy in my previous attempt at this my app was closed source.  He said that was allowed and maybe try to do it some way like this.  Think this will be ok?

Reply:
So how do you think buildbot will access your build artifact here? From the Ether?

And checking them in this repo is NOT an allowable solution either.

Reply:
ok I was wondering about that.  I guess I'll make a github repo of my build directory that this manifest can reference.  I think I'm back to the drawing board here for a little bit while I test it out and stuff.

If you happen to know of a flathub app that's closed source and a good thing to mimic, I'd appreciate it.  

Else I'll just get my build dir into a public github repo and change this manifest to reference it's url.

Thanks for helpin' me out here.  I just don't know how this stuff is done yet :)  I'll get there !